ext/standard/tests/streams/bug46024.phpt

```diff
@@ -12,7 +12,7 @@ if (getenv('SKIP_ASAN')) die("skip Test may leak");
 $php = realpath(getenv('TEST_PHP_EXECUTABLE'));
```

Comment:
`$php` is not needed then, also check all `TEST_PHP_EXECUTABLE`, the SKIPIF above needs to be updated too